packages/jest-config/src/__tests__/normalize.test.ts

```diff
@@ -46,8 +47,9 @@ const findNodeModule = jest.fn(name => {
 // Windows uses backslashes for path separators, which need to be escaped in
 // regular expressions. This little helper function helps us generate the
 // expected strings for checking path patterns.
-function joinForPattern() {
-  return Array.prototype.join.call(arguments, escapeStrForRegex(path.sep));
+function joinForPattern(...args) {
+  // eslint-disable-next-line prefer-rest-params
+  return Array.prototype.join.call(args, escapeStrForRegex(path.sep));
```

Comment:
since you've made the refactor to reset args, isn't this just `return args.join(escapeStrForRegex(path.sep))` now?